tt_metal/impl/dispatch/command_queue_interface.hpp

```diff
@@ -182,7 +182,13 @@ class SystemMemoryManager {
         uint16_t channel = tt::Cluster::instance().get_assigned_channel_for_device(device_id);
         char* hugepage_start = (char*) tt::Cluster::instance().host_dma_address(0, mmio_device_id, channel);
         this->cq_sysmem_start = hugepage_start;
-        this->cq_size = tt::Cluster::instance().get_host_channel_size(mmio_device_id, channel) / num_hw_cqs;
+        char* cq_size_override_env = std::getenv("TT_METAL_CQ_SIZE_OVERRIDE");
```

Comment:
do we need this overrride env var?

Reply:
This is to make CQ smaller for better local stress testing (aka, make the value be 1048576 to make whole CQ size be 1MB. 750KB issue, 250KB completion).